CHANGELOG/textcolor.opacity.md

```diff
@@ -0,0 +1,3 @@
+## TextColor Opacity
+- Fixed a failure to render translucent `TextColor` correctly
+- Fixed the rendering of opaque emoji when `TextColor` is translucent (they will now be translucent, though still  use the font coloring)
```

Comment:
Also, is there consensus to switch to the changelog-directory approach? I haven't heard anything about that...

Reply:
Not yet, as there was no daily. But I will collapse these into CHANGELOG.md at the time I merge if we don't agree on this.

Reply:
Fair enough!